### ISSUE TITLE
[Suggestion] Have tabs to differentiate database types

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/dbhub.io#199]
![image](https://user-images.githubusercontent.com/7872218/235359767-11daeb83-94cb-49c6-bbe1-e9a5c0798409.png)

Due to my bad organisational skills, I seem to have accumulated a lot of private databases.  This means scrolling down to the live databases is a lengthy process.
(Can see the scrollbar on the right)

Would it be possible to have a tabbed display of databases?  [Private Standard] [Private Live] [Public Standard] [Public Live]

I'm guessing the '[Shared with me] [Shared by me]' could be on the end, again, rather than scrolling down.  Unsure how the page jump will be .. page 1 with no databases would give a short page, but then would jump if the 'Private Standard' (like mine) was a long list.

![image](https://user-images.githubusercontent.com/7872218/235359985-719f8d59-78af-4487-8b3d-46e3e2eb1756.png)

As it may not be clear, I'm suggesting these sort of tabs.

**Comment by justinclift** [comment]
Yeah, I also reckon the existing profile page layout is reaching its end of life.

It was ok as a starting point, but like yourself I have quite a few databases in my account now.  So it's getting to the point of having to scroll quite a lot to access things too.

We could probably try out a tabbed approach (and others?) in some not-deployed-to-production private branch.  Might be a good idea to see what options we have, in case there's something better for us than just tabs too.

**Comment by chrisjlocke** [comment]
> there's something better for us than just tabs

![image](https://user-images.githubusercontent.com/7872218/235364315-9ef50ea3-bf78-4f02-8893-0a1654ae78ad.png)

Guess you could try the dropdown combobox stylee
